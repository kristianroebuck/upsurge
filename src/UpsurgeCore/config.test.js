import getConfig from './config';

describe('getConfig', () => {
  it('should return a full set of configuration options', () => {
    const config = getConfig();

    expect(config).toMatchSnapshot();
  });
});
