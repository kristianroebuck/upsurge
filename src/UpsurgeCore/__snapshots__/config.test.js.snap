// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getConfig should return a full set of configuration options 1`] = `
Object {
  "awsKey": undefined,
  "awsRegion": "us-east-1",
  "bucket": undefined,
  "s3BaseUrl": "https://undefined.s3.amazonaws.com/",
  "signerUrl": undefined,
}
`;
